app/utils/transistor.server.ts

```diff
@@ -84,6 +84,15 @@ async function createEpisode({
   })
   const {upload_url, audio_url, content_type} = authorized.data.attributes
 
+  const totalEpisodes = await fetchTransitor<TransistorEpisodesJson>({
```

Comment:
Is there any way for us to avoid calling this API? It's just sending a lot of data 🤔

Maybe we can request the seasons first, then query the latest season. Then we'd know from the start what season and episode we should create.

Reply:
I am unsure how we can request the season, 
https://developers.transistor.fm/#episodes  doc seemed a bit confusing to me. 
I tried to get latest episode and pulled the season out of it, I think that will give us the latest season